### PR TITLE
Delete todo

### PR DESCRIPTION
**Pull request by mungsehari**
* Add delete todo feature

### CODE REVIEW

File: src/pages/dashboard/todo/helpers/delete.tsx [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+import { Tooltip } from "antd";
+import { RiLoader4Line } from "react-icons/ri";
+import useUserInfo from "../../../../hooks/useUserInfo";
+import todoService from "../../../../services/todo-service";
+import ConfirmationPopUp from "../../../../components/ui/confirmation-popup";
+
+interface TodoCardProps {
+  objectId: string;
+  refetchTodoList: () => {};
+}
```
Comment (codemitrayt) [inline src/pages/dashboard/todo/helpers/delete.tsx:12]: Change interface TodoCardProps -> DeleteTodoProps

File: src/pages/dashboard/todo/helpers/delete.tsx [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+  const { authToken } = useUserInfo();
+  const { isLoading, mutate } = useMutation({
+    mutationKey: ["delete"],
+    mutationFn: ({ data }: { data: { objectId: string } }) =>
+      todoService().deleteTodo({ data, authToken }),
+    onSuccess: () => {
+      refetchTodoList();
+    },
+    onError: (error) => {
+      console.error("ERROR :: delete Todo ::", error);
```
Comment (codemitrayt) [inline src/pages/dashboard/todo/helpers/delete.tsx:23]: handle backend response error.

File: src/pages/dashboard/todo/helpers/delete.tsx [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+    },
+    retry: false,
+  });
+  const handleDelete = () => {
+    mutate({ data: { objectId } });
+  };
+  return (
+    <>
```
Comment (codemitrayt) [inline src/pages/dashboard/todo/helpers/delete.tsx:31]: remove react fragment.

**Review by codemitrayt — COMMENTED** [review]
Please fix the changes as I mentions.

File: src/pages/dashboard/todo/helpers/delete.tsx (hunk 1 above)
Comment (rushi-mungse) [reply src/pages/dashboard/todo/helpers/delete.tsx:12]: done

**Pull request merged**